### ISSUE TITLE
Semantic token highlighting

### ISSUE THREAD

**Issue by charconstpointer** [primer/github-vscode-theme#326]
It seems like this theme does not support semantic highlighting https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide

`Github Theme` with `"gopls": { "ui.semanticTokens": true }`

<img width="464" alt="Screenshot 2022-09-24 at 00 51 31" src="https://user-images.githubusercontent.com/35968924/192066731-4630c3d4-777b-4b57-a999-ffb871245b38.png">

`Dark+` with `"gopls": { "ui.semanticTokens": true }`

<img width="440" alt="Screenshot 2022-09-24 at 00 51 24" src="https://user-images.githubusercontent.com/35968924/192066734-8a640a51-fc3c-4612-a994-f3f16cc43849.png">

`Dark+`  with `"gopls": { "ui.semanticTokens": false }`

<img width="434" alt="Screenshot 2022-09-24 at 00 51 08" src="https://user-images.githubusercontent.com/35968924/192066736-3deb2270-6a3f-414d-8831-1b00f7bbd6e6.png">

Will this be supported in the future?

**Comment by ShadiestGoat** [comment]
Yes I came here for the exact same reason - %v in go strings is not highlighted!

**Comment by ShadiestGoat** [comment]
I believe that there is actual support for semantic highlighting to some extent actually! On GitHub Dark Default, are differences between a regular string and %v, 
![image](https://user-images.githubusercontent.com/48590492/199555345-97535b9b-fdc7-4b49-947b-58d46cd0f743.png)

It is just very low contrast

**Comment by ShadiestGoat** [comment]
![image](https://user-images.githubusercontent.com/48590492/199556119-96a072d5-3a4b-42ec-9b2b-037511fb8c7a.png)
![image](https://user-images.githubusercontent.com/48590492/199556154-945dfea8-2c10-4bb8-8baa-934e8e63b150.png)

**Comment by soraros** [comment]
@simurai I think this issue is not closed with #337 alone.

**Comment by simurai** [comment]
> I think this issue is not closed with https://github.com/primer/github-vscode-theme/pull/337 alone.

Ohh.. ok. Let's reopen it for further improvements.
